uber/templates/groups/form.html

```diff
@@ -201,7 +201,7 @@
             <input type="text" name="website" value="{{ group.website }}" class="form-control"/>
         </div>
         <div class="col-sm-9 col-sm-offset-3">
-            <a href="{{ group.website }}">{{ group.website }}</a>
+            <a href="{% if not group.website.startswith('http') %}http://{% endif -%}{{ group.website }}">{{ group.website }}</a>
```

Comment:
In my testing, I was adding URLs like "www.example.com". But when I clicked the link it would take me to https://localhost:4443/uber/registration/www.example.com.